content/docs/07-reference/05-ai-sdk-errors/ai-invalid-tool-input-error.mdx

```diff
@@ -10,9 +10,7 @@ This error occurs when invalid tool input was provided.
 ## Properties
 
 - `toolName`: The name of the tool with invalid inputs
-- `toolArgs`: The invalid tool inputs
-- `message`: The error message
-- `cause`: The cause of the error
+- `toolInput`: The invalid tool inputs
```

Comment:
it still accepts `cause` and `messag`

https://github.com/vercel/ai/blob/63f9e9b45b2e6a3b53fe9097c817cbd7fd3882a4/packages/ai/src/error/invalid-tool-input-error.ts#L13-L18

Reply:
do not update `packages/ai/src/error/invalid-tool-input-error.ts`, the implementation is correct. Keep `message` and `cause` in the docs